### ISSUE TITLE
assertion at two successive identical coordinates.

### ISSUE THREAD

**Issue by ruddyscent** [pbugnion/gmaps#176]
I'm using Python 3.5.3 and Jupyter 4.3.0 on Ubuntu 16.04.3. When I add a layer with two successive identical coordinates,
```
fig = gmaps.figure()
layer = gmaps.symbol_layer([(0,0),(0,0)], fill_color='green', stroke_color='green', scale=2)
fig.add_layer(layer)
```
gmaps raises the following assertion.
```
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-8-cd6c77bbc160> in draw_path(df, trip_id)
     20     fig = gmaps.figure()
     21     layer = gmaps.symbol_layer([(0,0),(0,0)], fill_color='green', stroke_color='green', scale=2)
---> 22     fig.add_layer(layer)
     23     return fig
     24 

/opt/continuum/anaconda3/lib/python3.5/site-packages/gmaps/figure.py in add_layer(self, layer)
     57                 Create a layer with directions
     58         """
---> 59         self._map.add_layer(layer)
     60 
     61 

/opt/continuum/anaconda3/lib/python3.5/site-packages/gmaps/maps.py in add_layer(self, layer)
    149 
    150     def add_layer(self, layer):
--> 151         self.layers = tuple([l for l in self.layers] + [layer])
    152 
    153     @default("layout")

/opt/continuum/anaconda3/lib/python3.5/site-packages/traitlets/traitlets.py in __set__(self, obj, value)
    583             raise TraitError('The "%s" trait is read-only.' % self.name)
    584         else:
--> 585             self.set(obj, value)
    586 
    587     def _validate(self, obj, value):

/opt/continuum/anaconda3/lib/python3.5/site-packages/traitlets/traitlets.py in set(self, obj, value)
    572             # we explicitly compare silent to True just in case the equality
    573             # comparison above returns something other than True/False
--> 574             obj._notify_trait(self.name, old_value, new_value)
    575 
    576     def __set__(self, obj, value):

/opt/continuum/anaconda3/lib/python3.5/site-packages/traitlets/traitlets.py in _notify_trait(self, name, old_value, new_value)
   1137             new=new_value,
   1138             owner=self,
-> 1139             type='change',
   1140         ))
   1141 

/opt/continuum/anaconda3/lib/python3.5/site-packages/ipywidgets/widgets/widget.py in notify_change(self, change)
    396                 # Send new state to front-end
    397                 self.send_state(key=name)
--> 398         LoggingConfigurable.notify_change(self, change)
    399 
    400     #-------------------------------------------------------------------------

/opt/continuum/anaconda3/lib/python3.5/site-packages/traitlets/traitlets.py in notify_change(self, change)
   1174                 c = getattr(self, c.name)
   1175 
-> 1176             c(change)
   1177 
   1178     def _add_notifiers(self, handler, name, type):

/opt/continuum/anaconda3/lib/python3.5/site-packages/gmaps/maps.py in _calc_bounds(self, change)
    169             ]
    170             min_longitude, max_longitude =\
--> 171                 merge_longitude_bounds(longitude_bounds)
    172 
    173             self.data_bounds = [

/opt/continuum/anaconda3/lib/python3.5/site-packages/gmaps/bounds.py in merge_longitude_bounds(longitude_bounds_list)
    139         seg_start = bnd
    140         coverage += delta
--> 141         assert(coverage >= 0)
    142 
    143     # reversing the lng order in a gap gives us a valid bound

AssertionError: 
```

**Comment by pbugnion** [comment]
Thanks for raising an issue. Well spotted.

I think the error is [here](https://github.com/pbugnion/gmaps/blob/master/gmaps/bounds.py#L43). We have a special case for when there is a single point when we calculate the latitude and longitude bounds. What we actually want is a special case for when `max(latitude) - min(latitude)` is less than `EPSILON`.

**Comment by pbugnion** [comment]
Fixed by #204.
